tests/src/Mod/Part/App/TopoShapeMakeShape.cpp

```diff
@@ -70,12 +70,10 @@ TEST_F(TopoShapeMakeShapeTests, shapeVertex)
     TopoShape topoShape(vertexMaker.Vertex(), 1L);
     // Act
     TopoShape& result = topoShape.makeElementShape(vertexMaker, topoShape);
-    auto elements = elementMap(result);
     // Assert
-    EXPECT_EQ(elements.size(), 1);
-    EXPECT_EQ(elements.count(IndexedName("Vertex", 1)), 1);
-    EXPECT_EQ(elements[IndexedName("Vertex", 1)], MappedName("Vertex1;MAK;:H:4,V"));
-    EXPECT_EQ(getArea(result.getShape()), 0);
+    EXPECT_EQ(result.getElementMap().size(), 0);
+    EXPECT_EQ(result.countSubElements("Vertex"), 1);
+    EXPECT_EQ(result.countSubShapes("Vertex"), 1);
```

Comment:
This check looks more generic than the previous one.

Is the former not working anymore?

Reply:
It's actually a more accurate test of a single vertex, which does not create an elementMap.  This is pretty much the only stuff to test.  The getArea does work,  but it isn't testing anything related to the elementMap, so not really relevant.